Gemfile

```diff
@@ -36,7 +36,7 @@ gem 'jbuilder'
 # gem "bcrypt", "~> 3.1.7"
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍